meson.build

```diff
@@ -0,0 +1,50 @@
+project(
+    'oopetris',
+    'cpp',
+    license: 'MIT',
```

Comment:
How dare you choose the license for me here? 
![KappaGIF](https://user-images.githubusercontent.com/51532668/226203767-87cfe760-bcdf-4f3a-9c1a-5c62ce7e8b5c.gif)

Reply:
It was just my standard meson template, that I modified, you can remove that and then it has no license, but according to the kapp this is fine :thinking:

Reply:
Yes, MIT is totally fine for me 👍